assets/js/util/sanitize.test.js

```diff
@@ -29,3 +29,21 @@ describe( 'sanitizeHTML', () => {
 		} );
 	} );
 } );
+
+describe( 'unTrailingSlashIt', () => {
```

Comment:
Let's give this the correct name too:
```suggestion
describe( 'untrailingslashit', () => {
```